### PR TITLE
Swift/add exercises to config

### PR DESCRIPTION
**Pull request by wneumann**
Updated the Swift track's config.json to include the added concept exercises.

### CODE REVIEW

File: languages/swift/config.json [hunk 1]
```diff
@@ -5,7 +5,49 @@
...
+        "slug": "optionals",
+        "uuid": "9D0BB0AD-904C-4D61-9D2C-AE37913D5741",
+        "concepts": ["optionals"],
+        "prerequisites": ["basics", "numbers", "optionals"]
+      },
+      {
+        "slug": "strings",
+        "uuid": "082D7AEB-EE25-4217-BD6F-82532A65797A",
+        "concepts": ["strings"],
+        "prerequisites": ["basics", "arrays"]
```
Comment (ErikSchierboom) [inline languages/swift/config.json:48]: If there are is a prerequisite that is not `"basics"`, the latter can be omitted (see [this section](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics)):

```suggestion
        "prerequisites": ["arrays"]
```

File: languages/swift/config.json [hunk 2]
```diff
@@ -5,7 +5,49 @@
...
+        "slug": "numbers",
+        "uuid": "3BDAD781-47E1-493A-834D-CA6AC8361FFD",
+        "concepts": ["numbers"],
+        "prerequisites": ["basics"]
+      },
+      {
+        "slug": "optionals",
+        "uuid": "9D0BB0AD-904C-4D61-9D2C-AE37913D5741",
+        "concepts": ["optionals"],
+        "prerequisites": ["basics", "numbers", "optionals"]
```
Comment (ErikSchierboom) [inline languages/swift/config.json:42]: If there are is a prerequisite that is not `"basics"`, the latter can be omitted (see [this section](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics)). The `"optionals"` concept is also listed as a concept _and_ a prerequisite. Maybe you meant something like this?

```suggestion
        "prerequisites": ["numbers"]
```

Or possibly you have another prerequisite?

File: languages/swift/config.json [hunk 3]
```diff
@@ -5,7 +5,49 @@
...
+    "concept": [
+      {
+        "slug": "arrays",
+        "uuid": "b6c532c9-1e89-4fbf-8f08-27f5befb5bb8",
+        "concepts": ["arrays"],
+        "prerequisites": ["basics"]
+      },
+      {
+        "slug": "basics",
+        "uuid": "E6143DBC-895E-47AA-B42C-3034E1F872C2",
```
Comment (ErikSchierboom) [inline languages/swift/config.json:17]: FYI we usually generate UUID's using the https://www.uuidgenerator.net/version4 website, which uses lowercase. But I don't think it matters, as it is still unique, right @iHiD?

Comment (iHiD) [reply languages/swift/config.json:17]: I'd rather they were lowercase, yes. But we'll probably do a big pass/update at the end before launch anyway.

**Comment by wneumann** [comment]
Pushed an update that lowercased the UUID strings as well as addressed the issues with the redundant and incorrect prerequisites.

**Comment by ErikSchierboom** [comment]
@wneumann I don't know if you already knew this, but if a PR has an approval the PR submitter is free to merge the PR (it is usually done by the PR submitted as they will know if they are done with the PR).

**Comment by wneumann** [comment]
@ErikSchierboom Apparently I need to be granted write access, as I'm given the message "Only those with write access to this repository can merge pull requests."

**Comment by ErikSchierboom** [comment]
@wneumann Hmmm, that is weird, as you should have permissions due to you being listed in the [maintainers.json](https://github.com/exercism/swift/blob/master/config/maintainers.json) file in the swift repo. 

@iHiD Did we have a permissions issue sometimes that you could solve manually?

I'll merge this then.

**Comment by iHiD** [comment]
@wneumann For some reason you weren't in the correct GH team - I've added you now.

**Pull request merged**